Examples/Modules/boosted_diags/inputs_3d_slice

```diff
@@ -133,6 +133,7 @@ btd_pltfile.dz_snapshots_lab = 0.001
 btd_pltfile.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz rho
 btd_pltfile.format = plotfile
 btd_pltfile.buffer_size = 32
+btd_pltfile.write_species = 1
```

Comment:
Can be a follow-up PR:
Should this file also add `btd_openpmd.write_species = 1`?

Reply:
openpmd is already writing particles.